test/dynamo/test_misc.py

```diff
@@ -10876,13 +10876,18 @@ def cf(x):
             torch._check_is_size(u0)
             torch._check_is_size(u1)
             torch._check(u0 + u1 == 20)
+
+            y = 0
             if guard_size_oblivious(torch.sym_max(1, u0 + u1) == 20):
```

Comment:
can we test expressions with mutliple max, mins or nested seems like the code tries to handles those?
            for atom in (*expr.atoms(Max), *expr.atoms(Min)):  # type: ignore[has-type]